tasks/send-slack-notification/send-slack-notification.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.0.1"
```

Comment:
Same thing with version as the other task - should probably be 1.0.0.

Reply:
fixed